recommendations/models.py

```diff
@@ -1,8 +1,35 @@
 from django.db import models
+from django.contrib.postgres.fields import ArrayField
+from .choices import *
 
 
-# Create your models here.
 class Movie(models.Model):
-    name = models.CharField(max_length=30, unique=False)
+    name = models.CharField(max_length=30)
+    genre = models.CharField(max_length=13, choices=GENRES)
```

Comment:
I think you meant genre = 'ArrayField(models.CharField(max_length=13, choices=GENRES))'? Also a bit nitpicky but now I'm thinking this field should be renamed 'genres' to better show plurality

Reply:
I'm not sure if you're trying to keep the model simple but I think a keywords field would make searching by mood a lot easier

Reply:
they are supposed to be arrayfields! good catch :)
keywords field is intentional - it currently doesn't have anything to do with sentiment analysis though which i know you wanted to do